UnityProject/Assets/Scripts/whisper/WhisperMicrophoneHandler.cs

```diff
@@ -57,6 +65,20 @@ private async void OnRecordStop(AudioChunk recordedAudio)
 	{
 		_buffer = "";
 
+		var ToUesChatChannel = ChatChannel.Local;
+		if (InputManagerWrapper.GetKey(KeyCode.Semicolon)) ToUesChatChannel |= ChatChannel.Common;
+		if (InputManagerWrapper.GetKey(KeyCode.B)) ToUesChatChannel |= ChatChannel.Binary;
+		if (InputManagerWrapper.GetKey(KeyCode.U)) ToUesChatChannel |= ChatChannel.Supply;
+		//if (InputManagerWrapper.GetKey(KeyCode.Y)) ToUesChatChannel |= ChatChannel.CentComm; Conflicts with opening chat with Local Preselected
+		if (InputManagerWrapper.GetKey(KeyCode.C)) ToUesChatChannel |= ChatChannel.Command;
+		if (InputManagerWrapper.GetKey(KeyCode.E)) ToUesChatChannel |= ChatChannel.Engineering;
+		//if (InputManagerWrapper.GetKey(KeyCode.M)) ToUesChatChannel |= ChatChannel.Medical; //Conflicts with toggling STT (This very thing )
+		//if (InputManagerWrapper.GetKey(KeyCode.N)) ToUesChatChannel |= ChatChannel.Science; //Conflicts with toggle voice chat
```

Comment:
![bafkreigtavdm2yj4wqlq7e42jcuixlz4xwt3ugvxxbcqrwnxangqt7xsbe](https://github.com/user-attachments/assets/5927723b-1e17-4a8e-be95-b3b2064f81bd)

Reply:
Comments

Reply:
well explains why x channel is not Implemented

Reply:
this is a terrible hack/workaround

Reply:
private static ChatChannel CheckRadio(string playerInput, IChatInputContext context, out int specialCharCount)
	
	if (firstLetter == ';')
	
	ChannelsTags[secondLetter];
	
	```
	public static readonly Dictionary<char, ChatChannel> ChannelsTags = new Dictionary<char, ChatChannel>
	{
		{'b', ChatChannel.Binary},
		{'u', ChatChannel.Supply},
		```

Reply:
![GM-tXBnaYAAeBGE](https://github.com/user-attachments/assets/31196d89-1f9c-4df3-a9c5-d583c092a9b2)